cmd/watch.go

```diff
@@ -15,6 +15,11 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var (
+	ignores []string
```

Comment:
we need to change this variable elsewhere, but in english, "ignoring" is better as "ignores" is third-person.

doesn't really matter though to be honest.

Reply:
So, may I retain it as it is ? Its also not user facing message anyway :)

Reply:
yeah, it's fine!